Packs/Core/Triggers/Trigger_-_Suspicious_msiexec_execution.json

```diff
@@ -0,0 +1,47 @@
+{
+  "trigger_id": "7bf91ab431cc4c478c4c70099ae110ec",
+  "playbook_id": "Suspicious msiexec execution",
+  "suggestion_reason": "Recommended for 'Msiexec execution of an executable from an uncommon remote location without properties' and 'Msiexec execution of an executable from an uncommon remote location with a specific port' alerts",
+  "description": "This trigger is responsible for handling the 'Msiexec execution of an executable from an uncommon remote location with a specific port' and 'Msiexec execution of an executable from an uncommon remote location without properties' alerts via the 'Suspicious msiexec execution' playbook",
+  "trigger_name": "Suspicious msiexec execution",
```

Comment:
we should update the playbook name and trigger (also the ID after) because its not reflecting the alerts the playbook.
the name is too generic for MSI execution. the playbook handles uncommon remote WMI execution..

Reply:
Updated to match the alert name `Msiexec execution of an executable from an uncommon remote location`